magento/pwa-studio
# Fix typo in component import name; consolidate import

## Verification Steps

1. Navigate to a category.
2. Make sure "Add to Cart" button exists.
3. Click "Add to Cart" button.

## Description

This fixes two nitpicks:

* The `button` in the import was inconsistent with both the file name and component.
* Going up to the parent directory and back down is inconsistent and unnecessary. 

### Resolved issues:
1. [x] resolves magento/pwa-studio#3719: Fix typo in component import name; consolidate import
